config.py

```diff
@@ -2,14 +2,17 @@
 TOKEN = "CENSORED"
 
 # Rules 'n' settings
-game_stage = 0
+dynamic_config = "dynamic.json"
 max_participants = 40
+max_cc_per_user = 8
+season = "1"
 
 prefix = '!'
 universal_prefix = "#!003-88-6521"
 
-# List of specific files
+# Database settings
 database = 'database.db'
+max_channels_per_category = 50
```

Comment:
Does this need to be in the config? It's a limit set by Discord, and I see no reason for it to change, unless you want _less_ than 50 channels in a category, which really would be inefficient.

Reply:
![image](https://user-images.githubusercontent.com/33968886/41870674-f75b22f2-78bc-11e8-9164-34922523e333.png)


You're welcome.

Reply:
What do you mean by this? Is this solely for testing porpoises?

Reply:
For testing purposes AND for the possibility of Discord deciding to allow more channels in one category. Such numbers have changed before, it is not unlikely they will again.